.github/workflows/e2e_tests.yaml

```
@@ -2,7 +2,7 @@
 name: E2E Tests
 
 on: 
-  [push, pull_request_target]
+  []
```

actionlint (1.7.12)

[error] "on" section should not be empty

(syntax-check)